packages/demos/todomvc-fluent/src/module/sequences.ts

```diff
@@ -1,65 +1,64 @@
-  .action(actions.redirectToAll)
-);
+export const redirectToAll = sequence(s => s.action(actions.redirectToAll))
 
-export const changeNewTodoTitle = sequenceWithProps<{ title: string }>(s => s
-  .action(actions.changeNewTodoTitle)
-);
+export const changeNewTodoTitle = sequenceWithProps<{ title: string }>(s =>
+  s.action(actions.changeNewTodoTitle)
+)
 
-export const removeTodo = sequenceWithProps<{ uid: string }>(s => s
-  .action(actions.removeTodo)
-);
+export const removeTodo = sequenceWithProps<{ uid: string }>(s =>
+  s.action(actions.removeTodo)
+)
 
-export const toggleAllChecked = sequence(s => s
-  .action(actions.toggleAllChecked)
-);
+export const toggleAllChecked = sequence(s =>
+  s.action(actions.toggleAllChecked)
+)
 
-export const toggleTodoCompleted = sequenceWithProps<{ uid: string }>(s => s
-  .action(actions.toggleTodoCompleted)
-);
+export const toggleTodoCompleted = sequenceWithProps<{ uid: string }>(s =>
+  s.action(actions.toggleTodoCompleted)
+)
 
-export const clearCompletedTodos = sequence(s => s
-  .action(actions.clearCompletedTodos)
-);
+export const clearCompletedTodos = sequence(s =>
+  s.action(actions.clearCompletedTodos)
+)
 
-export const changeFilter = sequenceWithProps<{ filter: string }>(s => s
-  .action(actions.changeFilter)
-);
+export const changeFilter = sequenceWithProps<{ filter: string }>(s =>
+  s.action(actions.changeFilter)
+)
 
-export const submitNewTodo = sequence(s => s
-  .when(x => !!x.state.newTodoTitle)
-  .paths({
+export const submitNewTodo = sequence(s =>
+  s.when(x => !!x.state.newTodoTitle).paths({
     true: s => s.action(actions.addTodo, actions.clearTodoTitle),
-    false: s => s
+    false: s => s,
   })
-);
+)
 
-export const changeTodoTitle = sequenceWithProps<{ uid: string, title: string }>(s => s
-  .action(actions.changeTodoTitle)
-);
+export const changeTodoTitle = sequenceWithProps<{
+  uid: string
+  title: string
+}>(s => s.action(actions.changeTodoTitle))
 
-export const editTodo = sequenceWithProps<{ uid: string }>(s => s
-  .action(actions.editTodo)
-);
+export const editTodo = sequenceWithProps<{ uid: string }>(s =>
+  s.action(actions.editTodo)
+)
 
-export const abortEdit = sequenceWithProps<{ uid: string }>(s => s
-  .action(actions.abortEditingTodo)
-);
+export const abortEdit = sequenceWithProps<{ uid: string }>(s =>
+  s.action(actions.abortEditingTodo)
+)
 
-export const submitTodoTitle = sequenceWithProps<{ uid: string }>(s => s
-  .branch(actions.whenEditedTitle)
-  // .branch<{ true: {}, false: {} }>(x => {
-  //   const todo = x.state.todos.get(x.props.uid);
-  //   if (todo && todo.editedTitle) {
-  //     return x.path.true({});
-  //   }
-  //   return x.path.false({});
-  // })
-  .paths({
-    true: s => s.action(actions.updateTodoTitle),
-    false: s => s
-  })
-);
+export const submitTodoTitle = sequenceWithProps<{ uid: string }>(s =>
+  s
+    .branch(actions.whenEditedTitle)
+    // .branch<{ true: {}, false: {} }>(x => {
```

Comment:
do not relate to PR, but what about this commented block? cc @christianalfoni

Reply:
Removed it now :)